packages/refresh-theme/config/omeda-identity-x.js

```diff
@@ -1,114 +1,110 @@
+      submitProfile: 6271,
     },
-    actionSource: {
-      id: 125,
-      valueIds: {
-        default: 451143,
-        newsletterSignup: 451144,
-        comments: 451145,
-        contentGate: 451142,
+    behaviorAttributes: {
+      website: {
+        id: 127,
+        valueId: websiteBehaviorAttributeId,
       },
-    },
-    newsletterSignupType: {
-      id: 126,
-      valueIds: {
-        default: 451148,
-        pushdown: 451147,
-        inlineContent: 451150,
-        inlineSection: 451149,
-        footer: 451146,
+      actionSource: {
+        id: 125,
+        valueIds: {
+          default: 451143,
+          newsletterSignup: 451144,
+          comments: 451145,
+          contentGate: 451142,
+        },
       },
-    },
-    contentGateType: {
-      id: 128,
-      valueIds: {
-        default: 451151,
-        metered: 451152,
-        printPreview: 451153,
+      newsletterSignupType: {
+        id: 126,
+        valueIds: {
+          default: 451148,
+          pushdown: 451147,
+          inlineContent: 451150,
+          inlineSection: 451149,
+          footer: 451146,
+        },
+      },
+      contentGateType: {
+        id: 128,
+        valueIds: {
+          default: 451151,
+          metered: 451152,
+          printPreview: 451153,
+        },
       },
     },
-  },
-  appendPromoCodeToHook,
-  appendBehaviorToHook,
-  appendDemographicToHook,
-  onAuthenticationSuccessFormatter: (async ({ req, payload }) => {
-    // BAIL if omedaGraphQLCLient isnt available return payload.
-    if (!req.$omedaGraphQLClient) return payload;
-
-    const identityXOptInHooks = req.app.locals.site.getAsObject('identityXOptInHooks');
-    const omeda = req.app.locals.site.getAsObject('omeda');
-    if (identityXOptInHooks.onAuthenticationSuccess) {
-      const { autoSignupDeploymentTypes } = identityXOptInHooks.onAuthenticationSuccess;
+    appendPromoCodeToHook,
+    appendBehaviorToHook,
+    appendDemographicToHook,
+    onAuthenticationSuccessFormatter: (async ({ req, payload }) => {
+      // BAIL if omedaGraphQLCLient isnt available return payload.
+      if (!omedaConfig.omedaGraphQLClientProp || !onAuthenticationSuccess) return payload;
+      const { autoSignupDeploymentTypes } = onAuthenticationSuccess;
       const { user } = payload;
       const found = getAsArray(user, 'externalIds')
         .find(({ identifier, namespace }) => identifier.type === 'encrypted'
           && namespace.provider === 'omeda'
-          && namespace.tenant === omeda.brandKey);
+          && namespace.tenant === omedaConfig.brandKey);
 
       // BAIL if no encryptedCustomerId and return payload
       if (!found) return payload;
       const encryptedCustomerId = get(found, 'identifier.value');
 
       // Retrive the omeda customer
       const omedaCustomer = await getOmedaCustomerRecord({
-        omedaGraphQLClient: req.$omedaGraphQLClient,
+        omedaGraphQLClient: req[omedaConfig.omedaGraphQLClientProp],
```

Comment:
If you define this above, you can just pass it through here
```suggestion
        omedaGraphQLClient,
```